config/routes.rb

```diff
@@ -2799,34 +2799,24 @@
       }.freeze
     end
 
-    def action_for_collection(collection_name, verb)
-      "#{collection_name}##{API_ACTIONS[verb]}"
-    end
-
-    def create_api_route(verb, url, action)
-      public_send(verb, url, :to => action)
-    end
-
     Api::Settings.collections.each do |collection_name, collection|
-      collection.verbs.each do |verb|
-        if collection.options.include?(:primary)
-          create_api_route(verb, collection_name, action_for_collection(collection_name, verb))
-        end
+      controller collection_name, :path => collection_name do
```

Comment:
Generally yes, except where the convention in the file is seattle rb style.  Note, routes.rb' common things like the `get`, `post`, `match` methods are ok without parens as that's closer to the API you see everywhere else.  I think this is ok here to keep the same style.  I was just :trollface: